src/elements/frame-router.ts

```diff
@@ -133,8 +137,11 @@ class FrameRouterElement extends HTMLElement {
   }
 
   private _dispatchClientMessage(message: ClientToHost) {
+    const messageDetail: any = message.msg;
+    messageDetail.origin = this._currentClientId;
```

Comment:
I think I liked `clientId` better than origin. I'm a bit worried about the potential for confusion with the existing meaning of origin in the contest of domains.

Reply:
Yea, I had debated about the same thing.  I ended up this way just because origin sounds nicer, but I'll change it back to avoid the confusion.

Reply:
There's clearly an insufficient supply of not-already-taken words in programming. Maybe we should all learn Japanese or French so we can get a whole new free set to use.